video.ino

```diff
@@ -269,10 +278,22 @@ void sendScreenPixel(int x, int y) {
 
 // Send an audio acknowledgement
 //
-void sendPlayNote(int channel, int success) {
-	byte packet[] = {
+// void sendPlayNote(int channel, int success) {
+// 	byte packet[] = {
+// 		channel,
+// 		success,
+// 	};
+// 	send_packet(PACKET_AUDIO, sizeof packet, packet);	
+// }
+
+void sendChannelStatus(int channel, int status) {
+	
+  if (status == 1) channel_status |= 1UL << channel;
+  if (status == 0) channel_status &= ~(1UL << channel);;
+  
+  byte packet[] = {
 		channel,
-		success,
+		channel_status,
 	};
 	send_packet(PACKET_AUDIO, sizeof packet, packet);	
 }

```

Comment:
hey @HeathenUK  - when we met at The Cave and chatted about this audio stuff, I mentioned that I liked your status approach, and that I planned on adopting it in my code.

If you look thru my PR you'll see that I ended up going in a different direction, so I thought I'd explain why.

Long story short, this changes the meaning of the second response byte so existing code that's checking it after attempting to play a note would not be compatible.

Specifically the current BBC BASIC `SOUND` command implementation would need a fairly complex rewrite.  Without that there would be circumstances where BASIC would fail to correctly work out whether an attempt to play a note was successful or not.  I couldn't quite work out how to make BASIC work correctly with your approach.

I also felt that it's slightly too limited, as there's more state than just "on/off" that it would be useful to convey.

The original meaning of this second byte is, essentially, to give an indicator of the status of the last audio command, with the only supported command being "play note".  With that in mind, I decided to just keep that meaning - so existing code can be compatible.

The underlying notion though that it'd be good to let code on the z80 side of the system be able to know the status of each audio channel is a great one though, hence why I implemented a "status" command.